akka-actor/src/main/scala/akka/pattern/AskSupport.scala

```diff
@@ -284,11 +284,21 @@ object AskableActorRef {
   /**
    * INTERNAL API
    */
-  @InternalApi private[akka] def recipientTerminatedExcpetion(recipient: Any, message: Any, sender: ActorRef): AskTimeoutException = {
+  @InternalApi private[akka] def recipientTerminatedException(recipient: Any, message: Any, sender: ActorRef): AskTimeoutException = {
     new AskTimeoutException(s"Recipient [$recipient] had already been terminated. " +
       messagePartOfException(message, sender))
   }
 
+  /**
+    * INTERNAL API
+    *
+    * @deprecated
+    */
+  @InternalApi
+  @deprecated("use recipientTerminatedException instead") private[akka] def recipientTerminatedExcpetion(recipient: Any, message: Any, sender: ActorRef): AskTimeoutException = {
+    recipientTerminatedException(recipient, message, sender)
+  }
```

Comment:
That's internal API so no need to keep the old name.

Reply:
Fixed by adding a mima filter.